x/concentrated-liquidity/README.md

```diff
@@ -228,13 +228,50 @@ b) Having the front end round the tick's actual price to the nearest human reada
 
 One draw back of this implementation is the requirement to create many ticks that will likely never be used. For example, in order to create ticks at 10 cent increments for spot prices greater than _$10000_, a $exponentAtPriceOne$ value of -5 must be set, requiring us to traverse ticks 1-3600000 before reaching _$10,000_. This should simply be an inconvenience and should not present any valid DOS vector for the chain.
 
-### Scope of Concentrated Liquidity
+# Chosing an Exponent At Price One Value
 
-#### Concentrated Liquidity Module
+The creator of a pool is required to choose an exponenetAtPriceOne as one of the input parameters. As explained previously, this value determines how much the spot price increases or decreases when traversing ticks. The following equation will assist in selecting this value:
+
+$$exponentAtPriceOne=log_{10}(\frac{D}{P})$$
+
+$$P=(\frac{baseAssetInUSD}{quoteAssetInUSD})$$
+
+$$D=P-(\frac{baseAssetInUSD}{quoteAssetInUSD+desiredIncrementOfQuoteInUSD})$$
```

Comment:
I think it would be really cool that if a user provides a desired price increment that the exponent gets selected automatically! Otherwise if the user provides an exponent then it is assumed that they know what they are doing. I will create an issue for this

Reply:
https://github.com/osmosis-labs/osmosis/issues/4922